index.html

```diff
@@ -8,20 +8,33 @@
     </head>
     <body class="overflow-hidden">
         <header>
-            <nav class="flex w-full h-16 bg-black text-white items-center">
+            <nav class="flex w-full h-[8vh] bg-black text-white items-center bg-gradient-to-t from-gray-800 via-gray-900 to-black">
```

Comment:
Are we going to use a dropdown menu for mobile devices? if so then the responsiveness of the navbar height here is not necessary, also on large screens, the increased size of the navbar looks weird since the text and image size does not increase proportionally.

I think instead of trying to dynamically change the size based on the height of the screen, it would be best to have a fixed size, and then users can zoom the page in if it is too small for them.
The only responsiveness changes we should do is rearranging elements that do not fit on the screen (i.e. logo/nav links when the screen size is really small) or (i.e. decreasing ultra large font sizes like headers for smaller screens)
```suggestion
            <nav class="flex w-full h-16 bg-black text-white items-center bg-gradient-to-t from-gray-800 via-gray-900 to-black">
```

Reply:
Made a commit bbbbb7d2e0a8194acb7b147bb3725d98a0e2d2d5.

I think it would be better if we just implement a dropdown menu as otherwise we would have to make a separate content page. It would also be simpler if visitors can directly go to content pages.